.gitignore

```diff
@@ -122,6 +122,9 @@ packages/*/types
 .vscode
 
 !**/fixtures/main/node_modules
+
+!packages/node/test/e2e/fixture/deptree/**/node_modules
```

Comment:
note to self: I think we can exempt any `node_modules` in `packages/node/test/*/fixture` from ignore